src/callbacks/callbacks.jl

```diff
@@ -312,13 +319,41 @@ function save_to_disk_func(integrator)
         )
         if p.atmos.precip_model isa Microphysics1Moment
             ᶜq_rai⁰ =
```

Comment:
I would suggest coding only for the 0-moment microphysics. And just throwing an error if someone wants to use EDMFX with 1-moment precipitation. I don't think we have it figured out if/how we want to include rain in the sub-domains.